### PR TITLE
New package: kittycad.ZooModelingApp version 0.14.0

### LINKED ISSUE #138602: [Package Request]: Zoo Modeling App
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://zoo.dev/modeling-app/download
Publisher: Zoo Corporation
Package Name: Zoo Modeling App
Description: Modeling App is an open-source hardware design interface that allows you to generate CAD models, both by editing code and with point-and-click actions. Get professional-grade modeling and low-latency computation without a massive workstation. The infrastructure behind the Modeling App utilizes our Design API and streams video from our Geometry Engine running in remote GPUs.
Package Version: 0.14.0
Installer URL: https://dl.zoo.dev/releases/modeling-app/v0.14.0/msi/Zoo%20Modeling%20App_0.14.0_x64_en-US.msi

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
- Resolve: #138602

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335215 2024-02-10 00:44:14.932 [CLI ] Package hash verification failed. SHA256 in manifest [e8602370b40b4596ddb86e569a058f8ab4a5d1617490ec1a5b28c49155d30a66] does not match download [0c881a709058aaecaf5428aff80b7ad5c211802e2cd1c8a21ef153dbaa4c71a6]

(Automated response - build 720.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Trenly** [comment]
I get a different hash every time I download the package

**Comment by stephengillie** [comment]
[Policy] Needs-Author-Feedback
[Policy] Changes-Requested

File: manifests/k/kittycad/ZooModelingApp/0.14.0/kittycad.ZooModelingApp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: kittycad.ZooModelingApp
+PackageVersion: 0.14.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{911085BA-EB5C-45C1-A557-294FCFF0CAE6}'
+Installers:
+- InstallerUrl: https://dl.zoo.dev/releases/modeling-app/v0.14.0/msi/Zoo%20Modeling%20App_0.14.0_x64_en-US.msi
+  Architecture: x64
+  InstallerSha256: 0c881a709058aaecaf5428aff80b7ad5c211802e2cd1c8a21ef153dbaa4c71a6
```
Comment (stephengillie) [inline manifests/k/kittycad/ZooModelingApp/0.14.0/kittycad.ZooModelingApp.installer.yaml:12]:
```suggestion
  InstallerSha256: E8602370B40B4596DDB86E569A058F8AB4A5D1617490EC1A5B28C49155D30A66
```

**Comment by stephengillie** [comment]
> I get a different hash every time I download the package

Hopefully their update rate has slowed enough to get the package through our pipelines.

File: manifests/k/kittycad/ZooModelingApp/0.14.0/kittycad.ZooModelingApp.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/k/kittycad/ZooModelingApp/0.14.0/kittycad.ZooModelingApp.installer.yaml:12]:
```suggestion
  InstallerSha256: E8602370B40B4596DDB86E569A058F8AB4A5D1617490EC1A5B28C49155D30A66
```

Comment (stephengillie) [inline manifests/k/kittycad/ZooModelingApp/0.14.0/kittycad.ZooModelingApp.installer.yaml:12]:
```suggestion
  InstallerSha256: E8602370B40B4596DDB86E569A058F8AB4A5D1617490EC1A5B28C49155D30A66
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335215 2024-02-15 11:17:31.797 [CLI ] Package hash verification failed. SHA256 in manifest [e8602370b40b4596ddb86e569a058f8ab4a5d1617490ec1a5b28c49155d30a66] does not match download [b7939e1397ab7a9008c6c7ece1624de8035389b7076e472936060fccb06e2ac8]

(Automated response - build 757.)

**Comment by stephengillie** [comment]
This package might be updated too quickly to be suitable for this kind of distribution.

**Pull request closed without merging**